SQL/paradise_schema.sql

```diff
@@ -473,14 +489,14 @@ DROP TABLE IF EXISTS `watch`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `watch` (
-  `ckey` varchar(32) NOT NULL,
-  `reason` text NOT NULL,
+  `ckey` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `reason` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
   `timestamp` datetime NOT NULL,
-  `adminckey` varchar(32) NOT NULL,
-  `last_editor` varchar(32),
-  `edits` text,
+  `adminckey` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `last_editor` varchar(32) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `edits` mediumtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   PRIMARY KEY (`ckey`)
```

Comment:
No `ckey` inde on watchlist? Its queried any time any player joins the server so it would make sense imo

Reply:
PRIMARY KEY is always an index.
https://stackoverflow.com/questions/1071180/is-the-primary-key-automatically-indexed-in-mysql

Reply:
Ah yes, forgot about that. For some reason `watchlist` is the only table which doesnt use an ID col

Reply:
Reason: for most tables, you can have multiple rows with the same ckey. E.g. there can be multiple bans, notes, etc that have the same ckey. So you need something to distinguish them - an ID number. But watchlist can only ever have one entry for any given ckey... so it makes sense for that to be the primary key of the table. No need for an ID number.
Why don't tables like, say, admin, which can only have one entry per ckey... ditch their ID number and just have ckey as their primary key? I don't know. Structure wise they could. But I'm trying to avoid like.... restructuring the entire database. So I'm not changing that.
But equally, I'm not adding an unnecessary id col to tables that need it. And I'm making use of alternate primary keys where they exist.

Reply:
Could be something for down the line. I know (probably) MSO did this for /tg/
![image](https://user-images.githubusercontent.com/25063394/97104378-5cf96900-16ab-11eb-9cab-606a0e2b0b4a.png)

*NOTE: `ckey` and `byond_key` are different things (KEY: `Key with spaces and case`, CKEY: `keywithspacesandcase`)*